code/modules/antagonists/_common/antag_datum.dm

```diff
@@ -87,16 +87,19 @@ GLOBAL_LIST_EMPTY(antagonists)
 	hud.leave_hud(mob_override)
 	set_antag_hud(mob_override, null)
 
-// Handles adding and removing the clumsy mutation from clown antags. Gets called in apply/remove_innate_effects
+
+/// Handles adding and removing the clumsy mutation from clown antags. Gets called in apply/remove_innate_effects
 /datum/antagonist/proc/handle_clown_mutation(mob/living/mob_override, message, removing = TRUE)
-	var/mob/living/carbon/human/H = mob_override
-	if(H && istype(H) && owner.assigned_role == "Clown")
-		if(removing) // They're a clown becoming an antag, remove clumsy
-			H.dna.remove_mutation(CLOWNMUT)
-			if(!silent && message)
-				to_chat(H, span_boldnotice("[message]"))
-		else
-			H.dna.add_mutation(CLOWNMUT) // We're removing their antag status, add back clumsy
+	if(!ishuman(mob_override) || owner.assigned_role.type != /datum/job/clown)
```

Comment:
Any particular reason as to why you'd use `type != type` instead of `!istype()`, or with the helper `!is_clown_job()`?

Reply:
None that matters. The original check was `variable != string` and I first changed it to use type directly, and then converted it to the is_type helpers. I guess I miss this one, thanks for spotting it. I'll fix it soon.
Do comment on others if you find them.